scripts/generateAssetData/generateAssetData.ts

```diff
@@ -165,7 +165,7 @@ const generateAssetData = async () => {
   )
 }
 
-;(async () => {
+const main = async () => {
```

Comment:
Paranoia not having this as an IIAFE so we can ensure the two promises (asset data generation and related asset index generation) run in sequence, not in parallel